CMSgov/design-system
# Wnmgds 1236 linting updates

### Changed

**Will be submitting another PR next for stylelint changes**

Removed the eslint & stylelint packages
Removed the linting script and references to it from the cmsds script
Added new yarn script commands `lint:eslint` and `lint:stylelint`, use npx so they're hopefully straightforward
Updated `yarn lint` to run those two
Changed most 'errors' to 'warnings' across the board
Updated stylelint and eslint rules to hopefully be more useful, giving warnings for things that we should eventually fix/resolve
Removed **every single reference** to eslint-ignore and stylelint-ignore in our codebase
Made a couple small code updates to resolve linter warnings for a few small things
Updated eslint, stylelint and prettier packages to latest, moved them to the main package.json
Updated chromedriver so I could run yarn test as I'm on chrome 99 now, backwards compatible

### Removed

packages:
eslint-config-design-system
stylelint-config-design-system

## How to test

`yarn install` `yarn lint:eslint` `yarn lint:stylelint``yarn lint` + all the existing yarn/storybook/build commands ensure nothing broke, my testing shows everything as working.
